core.lua

```diff
@@ -1,27 +1,37 @@
 BuffDuty = LibStub("AceAddon-3.0"):NewAddon("BuffDuty", "AceConsole-3.0")
 local CHAT_COMMAND = "buffduty"
+local MESSAGE_COMMAND = "buffduty-msg"
 local defaults = {
     factionrealm = {
-        duties_cache = {}
+        duties_cache = {},
+        custom_messages = {}
     }
 }
 
 function BuffDuty:OnInitialize()
     self:init()
+
+    BuffDuty.Messages:Initialise()
 end
 
 function BuffDuty:init()
     self:RegisterChatCommand(CHAT_COMMAND, "Command")
+    self:RegisterChatCommand(MESSAGE_COMMAND, "CommandMessage")
 
     --Init DataBase
     self.db = LibStub("AceDB-3.0"):New("BuffDutyDB", defaults)
     BuffDuty.Cache.duties_cache = self.db.factionrealm.duties_cache
+    BuffDuty.Messages.custom_messages = self.db.factionrealm.custom_messages
+end
+
+function BuffDuty:OnEnable()
```

Comment:
What is OnEnable event?

Reply:
It's an Ace function, called when an Ace addon is enabled, which they are by default... I was just copying what they had in the AceDB example, however `Oninitialise` is probably more appropriate as `OnEnabled` seems more for subscribing to events (and unsubscribing in `OnDisabled`)

Reply:
I've removed OnEnable and put Messages:Load() into BuffDuty:init()